vmware-base/pom.xml

```diff
@@ -75,5 +75,11 @@
             <version>${cs.jaxws.version}</version>
             <type>pom</type>
         </dependency>
+        <dependency>
+            <groupId>com.cloud.com.vmware</groupId>
+            <artifactId>pbm</artifactId>
+            <version>${cs.vmware.api.version}</version>
+            <scope>compile</scope>
+        </dependency>
```

Comment:
this is called from both the hypervisor plugin and from the base lib. That does not seem like proper isolation.